### PR TITLE
feat: added ElasticBeanstalk worker server for pages revalidation

### PR DESCRIPTION
**Pull request by RomanBobrovskiy**
- added elb worker queue for pages revalidation
- updated variables names
- added uploading elb versions

### CODE REVIEW

File: src/cdk/constructs/CheckExpirationLambdaEdge.ts [hunk 1]
```diff
@@ -26,17 +23,15 @@ export class CheckExpirationLambdaEdge extends Construct {
...
-    const { bucketName, bucketRegion, ebAppDomain, nodejs, buildOutputPath, cacheConfig } = props
+    const { nodejs, buildOutputPath, cacheConfig, renderWorkerQueueUrl } = props
     super(scope, id)
 
     const nodeJSEnvironment = NodeJSEnvironmentMapping[nodejs ?? ''] ?? NodeJSEnvironmentMapping['20']
     const name = 'checkExpiration'
 
     buildLambda(name, buildOutputPath, {
       define: {
-        'process.env.S3_BUCKET': JSON.stringify(bucketName),
```
Comment (RomanBobrovskiy) [inline src/cdk/constructs/CheckExpirationLambdaEdge.ts:37]: not in use, were copied from another lambda. 
Actual lambda code updates will go in separate PR

File: src/cdk/constructs/CheckExpirationLambdaEdge.ts [hunk 2]
```diff
@@ -55,12 +50,5 @@ export class CheckExpirationLambdaEdge extends Construct {
     })
 
     logGroup.grantWrite(this.lambdaEdge)
-
-    const policyStatement = new iam.PolicyStatement({
```
Comment (RomanBobrovskiy) [inline src/cdk/constructs/CheckExpirationLambdaEdge.ts:59]: no need to access s3 bucket

File: src/cdk/constructs/RenderWorkerDistribution.ts [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+        // SQS Daemon Configuration
+        {
+          namespace: 'aws:elasticbeanstalk:sqsd',
+          optionName: 'WorkerQueueURL',
+          value: this.workerQueue.queueUrl
+        },
+        {
+          namespace: 'aws:elasticbeanstalk:sqsd',
+          optionName: 'HttpPath',
+          value: '/api/revalidate'
```
Comment (RomanBobrovskiy) [inline src/cdk/constructs/RenderWorkerDistribution.ts:281]: endpoint should be added in separate PR

**Review by dmitryhruzin — APPROVED** [review]
